static/locales/en-US.yaml

```diff
@@ -0,0 +1,314 @@
+    Theme Settings: Theme Settings
+        Match Top Bar with Main Color: Match Top Bar with Main Color
+        Base Theme: Base Theme
+            Black: Black
+            Dark: Dark
+            Light: Light
+        Main Color Theme: Main Color Theme
+            Red: Red
+            Pink: Pink
+            Purple: Purple
+            Dark Purple: Dark Purple
+            Indigo: Indigo
+            Blue: Blue
+            Light Blue: Light Blue
+            Cyan: Cyan
+            Teal: Teal
+            Green: Green
+            Light Green: Light Green
+            Lime: Lime
+            Yellow: Yellow
+            Amber: Amber
+            Orange: Orange
+            Deep Orange: Deep Orange
+        Secondary Color Theme: Secondary Color Theme
+        #* Main Color Theme
+    Player Settings: Player Settings
+        Force Local Backend for Legacy Formats: Force Local Backend for Legacy Formats
+        Autoplay Videos: Autoplay Videos
+        Proxy Videos Through Invidious: Proxy Videos Through Invidious
+        Autoplay Playlists: Autoplay Playlists
+        Enable Theatre Mode by Default: Enable Theatre Mode by Default
+        Default Volume: Default Volume
+        Default Playback Rate: Default Playback Rate
+        Default Video Format: Default Video Format
+            Dash Formats: Dash Formats
+            Legacy Formats: Legacy Formats
+            Audio Formats: Audio Formats
+        Default Quality: Default Quality
+            Auto: Auto
+            144p: 144p
+            240p: 240p
+            360p: 360p
+            480p: 480p
+            720p: 720p
+            1080p: 1080p
+            1440p: 1440p
+            4k: 4k
+            8k: 8k
+
+About: About
+    #On About Page
+    #& About
+    'This software is FOSS and released under the GNU Public License v3+.': >
+    'This copylefted software is freely licensed GPLv3+.'
+
+    'Found a bug? Want to suggest a feature? Want to help out? Check out our GitHub page. Pull requests are welcome.': >-
+      'Found a bug? Want to suggest a feature? Want to help out? Check out our GitHub
+      page. Pull requests welcome.'
+
+    Thank you very much to the People and Projects that make FreeTube possible!: >
+    Thank you very much to the People and Projects that make FreeTube possible!
+
+    'Want to chat? Join our Riot / Matrix Server . Please check the rules before joining.': >
+    'Want to chat? Join our Riot / Matrix Server . Please check the rules before joining.'
+
+    'Looking for help? Check out our Wiki page.': 'Looking for help? Check out our Wiki page.'
+
+    Check out our Firefox extension!: Check out our Firefox extension!
+
+    'If you enjoy using FreeTube, consider donating via Liberapay or through our Bitcoin address.': >
+    'If you enjoy using FreeTube, consider donating via Liberapay or through our Bitcoin address.'
+
+    'BTC: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS': 'BTC: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS'
```

Comment:
The Bitcoin Address can be removed as this won't need to be translated.  Not sure if `BTC` needs to be translated either but I'd want a second opinion on it.